e2e/accounts/accounts_test.go

```diff
@@ -252,12 +252,11 @@ func (s *AccountsTestSuite) TestSelectedAccountOnRestart() {
 	s.False(whisperService.HasKeyPair(pubKey1), "identity should be removed, but it is still present in whisper")
 
 	// stop node (and all of its sub-protocols)
-	nodeConfig, err := s.Backend.NodeManager().NodeConfig()
+	preservedNodeConfig, err := s.Backend.NodeManager().NodeConfig()
```

Comment:
Maybe just `nodeConfig`? It's used only in this test.

Reply:
Ok, but we'll lose context.

Reply:
@JekaMas Am not sure i understand the area of context?